server/services/config/conditions/v2/conditions.ts

```diff
@@ -40,6 +41,16 @@ export const modifyAdditionalConditions = (someConditions: AdditionalConditions)
       appointmentProfessions.map((condition, index) => (index > 0 ? ` ${condition}` : condition))
     )
   }
+
+  if (!isEmpty(conditions?.REPORT_TO)) {
+    const { reportingTime, reportingDaily, ...rest } = conditions?.REPORT_TO
+
+    conditions.REPORT_TO = {
+      reportingTime: reportingTime ? `at ${reportingTime}` : reportingDaily,
```

Comment:
prefix "at" or default to daily if not provided - this is used to render the full report